lib/bosh/NetworkSegmentIndex.js

```diff
@@ -48,8 +47,8 @@ NetworkSegmentIndex.capacity = calculateCapacity;
 
 module.exports = NetworkSegmentIndex;
 
-function calculateCapacity(subnet) {
-  let infrastructure = config.director.infrastructure;
+function calculateCapacity(subnet, config) {
+  let infrastructure = config.infrastructure;
```

Comment:
Cannot put in BoshDirectorClinet becuse of cyclic dependancy. So implementing in NetworkSegmentIndex itself.